File: CAPITULO 01/C++/ex12.cpp
#include <stdio.h>
#include <math.h>
int main()
{
	int num1,num2,a,b;
	printf("Digite um numero inteiro e positivo :");
	scanf("%d%*c",&num1);
	printf("Digite um numero inteiro e positivo :");
	scanf("%d%*c",&num2);
	a = pow(num1,num2);
	b = pow(num2,num1);
	printf("O numero %d elevado a %d eh = %.2d",num1,num2,a);
	printf("\n");
	printf("O numero %d elevado a %d eh = %.2d",num2,num1,b);
	getchar();
	return 0;
}

File: CAPITULO 01/Java/ex12.java
import java.io.*;
import java.util.*;
import java.math.*;

public class ex12{
	public static void main(String[]args){
		double num1,num2,a,b;
		Scanner dado;
		dado =new Scanner(System.in);
		System.out.println("Digite um numero inteiro e positivo maior do que 0 :");
		num1 = dado.nextDouble();
		System.out.println("Digite outro numero inteiro e positivo maior do que 0 :");
		num2 = dado.nextDouble();
		a = Math.pow(num1,num2);
		b = Math.pow(num2,num1);
		System.out.println("Numero "+num1+" elevado a "+num2+" = "+a+
			"\nNumero "+num2+ "elevado "+num1+" = "+ b);
	}
}